runtime/queries/sql/highlights.scm

```diff
@@ -1,343 +1,391 @@
+(invocation
+  (object_reference
+    name: (identifier) @function.method))
 
 [
+  (keyword_gist)
   (keyword_btree)
   (keyword_hash)
-  (keyword_gist)
   (keyword_spgist)
   (keyword_gin)
   (keyword_brin)
-
-  (cast)
-  (group_concat)
-  (invocation)
+  (keyword_array)
 ] @function.builtin
-  
-(table_reference
-  name: (identifier) @namespace)
 
-(relation
-  table_alias: (identifier) @variable.parameter)
-  
-(field
+(object_reference
   name: (identifier) @variable.other.member)
-  
+
+(relation
+  alias: (identifier) @variable.parameter)
+
 (field
-  table_alias: (identifier) @variable.parameter
   name: (identifier) @variable.other.member)
 
-(comment) @comment
+(term
+  alias: (identifier) @variable.parameter)
 
-[
-  "("
-  ")"
-] @punctuation.bracket
+(term
+   value: (cast
+    name: (keyword_cast) @function.builtin
+    parameter: [(literal)]?))
 
-[
-  ";"
-  ","
-  "."
-] @punctuation.delimiter
+(literal) @string
+(comment) @comment.line
+(marginalia) @comment.block
 
-(binary_expression
-  operator: _ @operator)
+((literal) @constant.numeric.integer
+ (#lua-match? @constant.numeric.integer "^%d+$"))
 
-(unary_expression
-  operator: _ @operator)
+((literal) @constant.numeric.float
+(#lua-match? @constant.numeric.float "^[-]?%d*\.%d*$"))
```

Comment:
Same here - I think the regex from the old highlights should work in place of this one (I think the `%d` is lua-regex-specific)